roofit/roofitcore/src/RooNLLVarNew.cxx

```diff
@@ -287,8 +275,19 @@ RooNLLVarNew::fillNormSetForServer(RooArgSet const & /*normSet*/, RooAbsArg cons
    return nullptr;
 }
 
-double RooNLLVarNew::getFinalValAfterOffsetting(ROOT::Math::KahanSum<double> &&result) const
+void RooNLLVarNew::enableOffsetting(bool flag) {
+   _doOffset = flag;
+   _offset = {};
+}
+
+double RooNLLVarNew::finalizeResult(ROOT::Math::KahanSum<double> &&result, double weightSum) const
 {
+   // If part of simultaneous PDF normalize probability over
+   // number of simultaneous PDFs: -sum(log(p/n)) = -sum(log(p)) + N*log(n)
```

Comment:
I don't know why we are adding this term that does not depend on parameters, are there reasons to keep it ?

Reply:
That's true, it doesn't change the result. But it was there in the old test statistics and there are some unit tests that explicitly check that this normalization is done. For now, I try to be conservative and reproduce the old behavior as closely as possible